src/components/cards/ClientTaskCard.tsx

```diff
@@ -32,7 +33,7 @@ export const ClientTaskCard = ({
         <Stack direction="row" columnGap={8} alignItems="center" justifyContent="space-between">
           <Stack sx={{ width: '100%', cursor: 'pointer' }} direction="column" onClick={() => handleRouteChange()}>
             <Typography variant="sm">{task?.title}</Typography>
-            {/* <Typography variant="bodySm">{task?.body}</Typography> */}
+            <Tapwrite content={task?.body || ''} getContent={() => {}} readonly />
```

Comment:
Why is `getContent` an empty function? Is there a way we could make getContent prop optional in Tapwrite instead?

Reply:
Yes sure. We can do that but it needs some work in Tapwrite. I will do it in the next PR? @rrojan